src/mono/mono/eventpipe/ep-rt-mono.h

```diff
@@ -982,6 +1216,57 @@ ep_rt_thread_get_or_create (void)
 #endif
 }
 
+static
+inline
+int32_t
+ep_rt_mono_thread_sleep (uint32_t ms, bool alertable)
+{
+	gboolean alerted = false;
+#ifdef EP_RT_MONO_USE_STATIC_RUNTIME
+	if (alertable)
+		return (int32_t)mono_thread_info_sleep (ms, &alerted);
+	else
+		return (int32_t)mono_thread_info_sleep (ms, NULL);
+#else
+	if (alertable)
+		return (int32_t)ep_rt_mono_func_table_get ()->ep_rt_mono_thread_info_sleep (ms, &alerted);
+	else
+		return (int32_t)ep_rt_mono_func_table_get ()->ep_rt_mono_thread_info_sleep (ms, NULL);
+#endif
+}
+
+static
+inline
+bool
+ep_rt_mono_thread_yield (void)
+{
+#ifdef EP_RT_MONO_USE_STATIC_RUNTIME
+	return (bool)mono_thread_info_yield ();
+#else
+	return (bool)ep_rt_mono_func_table_get ()->ep_rt_mono_thread_info_yield ();
+#endif
```

Comment:
Is there some way we can make it harder to introduce differences in this pattern?

Reply:
Can make ep_rt_mono_func_table_get undefined when using static runtime linkage and passing NULL for function table in call to ep_init.